lib/src/dbs/node.rs

```diff
@@ -19,7 +19,7 @@ pub struct ClusterMembership {
 // This struct is meant to represent a timestamp that can be used to partially order
 // events in a cluster. It should be derived from a timestamp oracle, such as the
 // one available in TiKV via the client `TimestampExt` implementation.
-#[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize, PartialOrd, Hash, Store)]
+#[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize, PartialOrd, Hash, Store, Default)]
```

Comment:
What do we use Timestamp's `Default` for?

Reply:
Laziness 🤣 There was a reason, I believe. But realistically it won't be used that much. I will try to remove it and find out.

Reply:
Turns out it is required by the revisioned annotation